scout/server/blueprints/variant/templates/variant/variant.html

```diff
@@ -275,24 +275,29 @@ <h6>No matching causative or managed variants</h6>
               </span>
             </td>
             <td>
+              <!-- 🚧 untested code ahead 🚧 -->
+              {% if not variant.str_repid %}
```

Comment:
This works, so no worries, but actually better to check the category like before: we have started using hgnc_id for strs, and may end up deprecating repid at some point.